lib/backend/etcd/etcd.go

```diff
@@ -134,19 +134,21 @@ func (c *EtcdClient) EnsureCalicoNodeInitialized(node string) error {
 	// The confd agent used for BIRD configuration monitors certain
 	// directories and doesn't handle the non-existence of these directories
 	// very well, so create the required directories.
-	if err := c.ensureDirectory("/calico/v1/ipam/v4/pool"); err != nil {
-		return err
-	} else if err = c.ensureDirectory("/calico/v1/ipam/v6/pool"); err != nil {
-		return err
-	} else if err = c.ensureDirectory("/calico/bgp/v1/global/custom_filters/v4"); err != nil {
-		return err
-	} else if err = c.ensureDirectory("/calico/bgp/v1/global/custom_filters/v6"); err != nil {
-		return err
-	} else if err := c.ensureDirectory("/calico/ipam/v2/host/" + node + "/ipv4/block"); err != nil {
-		return err
-	} else if err = c.ensureDirectory("/calico/ipam/v2/host/" + node + "/ipv6/block"); err != nil {
-		return err
+	dirs := []string{
+		"/calico/v1/ipam/v4/pool",
+		"/calico/v1/ipam/v6/pool",
+		"/calico/bgp/v1/global/custom_filters/v4",
+		"/calico/bgp/v1/global/custom_filters/v6",
+		"/calico/ipam/v2/host/" + node + "/ipv4/block",
+		"/calico/ipam/v2/host/" + node + "/ipv6/block",
+	}
+
+	for _, d := range dirs {
```

Comment:
I guess the definition of dirs could even be inlined here.  Would that be more or less idiomatic?